crates/ide-completion/src/render.rs

```diff
@@ -595,6 +621,206 @@ mod tests {
         }
     }
 
+    #[test]
+    fn set_struct_type_completion_info() {
```

Comment:
What do you think of the tests? I will add similar tests for the remaining cases as well if these look good.

Reply:
Instead of doing mere `check` tests here, let's make use of `check_relevance` instead, as that is what we are mainly interested in checking here (those are used a bit further down in this module)